doc/ondemand.md

```diff
@@ -714,26 +714,16 @@ We expect that the On Demand approach has many of the performance benefits of th
 
 ### Limitations of the On Demand Approach
 
-The On Demand approach has  some limitations:
+The On Demand approach has some limitations:
 
-* Because it operates in streaming mode, you only have access to the current element in the JSON document. Furthermore, the document is traversed in order so the code is sensitive to the order of the JSON nodes in the same manner as an event-based approach (e.g., SAX).
+* Because it operates in streaming mode, you only have access to the current element in the JSON document. Furthermore, the document is traversed in order so the code is sensitive to the order of the JSON nodes in the same manner as an event-based approach (e.g., SAX). (The one exception to this is field lookup, which is more *performant* when the order of lookups matches the order of fields in the document, but which will still work with out-of-order fields, with a performance hit.)
```

Comment:
Ok, so find_field is not going to be in our documentation for the time being, correct? (Just checking.)